tests/performance-test/grafana/grafana-deploy.yml

```diff
@@ -19,13 +19,36 @@ spec:
          image: grafana/grafana
          ports:
            - containerPort: 3000
+         env:
+           - name: CACERT
+             valueFrom:
+              secretKeyRef:
+                name: elasticsearch
+                key: admin-ca
+           - name: CLIENTCERT
+             valueFrom:
+              secretKeyRef:
+                name: elasticsearch
+                key: admin-cert
+           - name: CLIENTKEY
+             valueFrom:
+              secretKeyRef:
+                name: elasticsearch
+                key: admin-key
          volumeMounts:
            - name: grafana-config
              mountPath: /etc/grafana/grafana.ini
              subPath: grafana.ini
+           - name: datasources
```

Comment:
:+1: You told me about this the other day. Much cleaner, cool.

Reply:
Ooooh nice!